main.js

```diff
@@ -71,26 +76,38 @@ function generateHexCode(){
 
 //DOM- INNERHTML/TEXT, anything we see on the page 
 function generatePalette(currentPalette) {
-    var newPalette = [];
+    var newHexColors = [];
     for (var i = 0; i < currentPalette.hexColors.length; i++) {
         if (!currentPalette.hexColors[i].isLocked) {
             var hex = {
                 hexCode: generateHexCode(),
                 isLocked: false,
                 id: `color${i+1}`
             };
-            newPalette.push(hex)
+            newHexColors.push(hex)
         } else {
-            newPalette.push(currentPalette.hexColors[i])
+            newHexColors.push(currentPalette.hexColors[i])
         }
     } 
     var updatedPalette = {
-        hexColors: newPalette,
+        hexColors: newHexColors,
         id: Date.now()
     }
     return updatedPalette
 }
 
+function dispayMinis(){
+    savedPaletteBoxCont.innerHTML += `
+    <div class="saved-palette-box-container">
+        <div class="mini-box" style="background-color: ${currentPalette.hexColors[0].hexCode}"></div>
+        <div class="mini-box" style="background-color: ${currentPalette.hexColors[1].hexCode}"></div>
+        <div class="mini-box" style="background-color: ${currentPalette.hexColors[2].hexCode}"></div>
+        <div class="mini-box" style="background-color: ${currentPalette.hexColors[3].hexCode}"></div>
+        <div class="mini-box" style="background-color: ${currentPalette.hexColors[4].hexCode}"></div>
+        </div>
+    `
```

Comment:
can we remove this space here ?

Reply:
like move the back tick on line 108 to 107?